src/main/java/com/exasol/exaoperation/plugin/Plugin.java

```diff
@@ -87,14 +89,50 @@ public String getFileName() {
      */
     // [impl->dsn~installing-plug-ins~1]
     public ExecResult install() {
-        return runScript("install");
+        return callFunction("install");
     }
 
-    public ExecResult runScript(final String method) {
+    /**
+     * Call one of the plugin's functions.
+     *
+     * @param method name of the function to call. plugins should handle this case-insensitive
+     * @return result of the function call
+     */
+    public ExecResult callFunction( final String method) {
+        return callFunction( method, "" );
```

Comment:
Spaces around parameters feels inconsistent, could you please remove them? 
Here and all other method definitions.